.github/workflows/release.yml

```diff
@@ -0,0 +1,81 @@
+---
+name: "TER deployment"
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  set-matrix:
+    runs-on: ubuntu-20.04
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - id: set-matrix
+        run: |
+          JSON=$(cat ./composer.json)
+          echo "::set-output name=${JSON//'%'/'%25'}"
+
+  # The actual TER upload job
+  ter-release:
+    needs: set-matrix
+    name: "TYPO3 TER release"
+
+    if: startsWith(github.ref, 'refs/tags/')
+
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php-versions: [ 7.4 ]
+        composer-data: ${{fromJson(needs.set-matrix.outputs.matrix)}}
+
+    env:
+      TYPO3_API_TOKEN: ${{ secrets.TYPO3_API_TOKEN }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Check tag
+        run: |
+          if ! [[ ${{ github.ref }} =~ ^refs/tags/[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}$ ]]; then
+            exit 1
+          fi
+
+      - name: Get version
+        id: get-version
+        run: echo ::set-output name=version::${GITHUB_REF/refs\/tags\//}
+
+      - uses: jungwinter/split@v1
+        id: split
+        with:
+          msg: ${{ matrix.composer-data.name}}
+          separator: '/'
+
+      - name: Remove files
+        run: >
+          rm -rf CODE_OF_CONDUCT.md Tests codeception.yml phpcs.xml .github .gitlab .editor .Build .gitattributes
+          .editorconfig .gitignore .php_cs.php Resources/Private/.eslintignore Resources/Private/.eslintrc.json
+          Resources/Private/.prettierrc.js Resources/Private/package.json Resources/Private/stylelint.config.js"
+
+      - name: Get comment
+        id: get-comment
+        run: |
+          readonly local comment=$(git tag -n10 -l ${{ steps.get-version.outputs.version }} | sed "s/^[0-9.]*[ ]*//g")
+          if [[ -z "${comment// }" ]]; then
+            echo ::set-output name=comment::Released version ${{ steps.get-version.outputs.version }} of ${{ steps.split.outputs._1 }}
+          else
+            echo ::set-output name=comment::$comment
+          fi
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+          extensions: intl, mbstring, json, zip, curl
```

Comment:
Please let's set the Composer version (2) explicitly (like [in the CI workflow](https://github.com/TYPO3-Documentation/tea/blob/main/.github/workflows/ci.yml#L42)).